static/modes/asm-mode.js

```diff
@@ -119,5 +119,9 @@ function definition() {
     };
 }
 
+var def = definition();
+
 monaco.languages.register({id: 'asm'});
 monaco.languages.setMonarchTokensProvider('asm', definition());
```

Comment:
Can you pass `def` here too?

Reply:
`def` is now directly passed to `setMonarchTokensProvider` but I am not sure if this is what you meant with you comment. Maybe you could elaborate your comment a bit, if this is the case.

Reply:
Yup! We mostly want to avoid calculating the definition twice here. Thanks!